### ISSUE TITLE
Training loss plots

### ISSUE THREAD

**Issue by ohadvb** [jik876/hifi-gan#59]
First of all, thank you for sharing your repo, and congratulations on your excellent results and lovely paper!

I've been trying to train your model (with slight alterations to fit some of my existing code) and I noticed a strange phenomenon, the deep feature loss seems to steadily increase until it reaches some plateau around ~12 (even when using the pretrained universal model).

I wonder if you could maybe share the plots for the various losses during training if you have those? 

thanks in advance!

**Comment by jik876** [comment]
Thank you very much. And sorry for the late reply.
The loss can increase, while training the discriminator. Since experiments logs were archived, it would be difficult to share the plots, 
but referring to our experiments, the phenomenon is not a problem.
